src/happiness_level.py

```diff
@@ -1,5 +1,85 @@
 import numpy as np
+import pandas as pd
+from typing import List, Union
+from outcomes import Result
 
-def happiness_level(s: np.array) -> float:
-    return .0
 
+class HappinessLevel:
+    """
+    Happiness level, ranges from 0 to -inf
+    """
+
+    def __init__(
+        self, preferences: Union[np.ndarray, pd.DataFrame], winner: Union[str, Result]
```

Comment:
Shouldn't `preferences` be `VotingArray`?

Reply:
Makes sense, it now only accepts `VotingArrays`, getting the column names using the `.to_pandas()` method